src/test/ui/cfg-attr-syntax-validation.stderr

```diff
@@ -40,10 +40,10 @@ error[E0537]: invalid predicate `a`
 LL | #[cfg(a())] //~ ERROR invalid predicate `a`
    |       ^^^
 
-error: literal in `cfg` predicate value must be a string
+error[E0565]: unsupported literal
   --> $DIR/cfg-attr-syntax-validation.rs:22:11
    |
-LL | #[cfg(a = 10)] //~ ERROR literal in `cfg` predicate value must be a string
+LL | #[cfg(a = 10)] //~ ERROR unsupported literal
```

Comment:
This change feels like a slight step backwards, from a very specific diagnostic towards a more general one... Shouldn't be a blocker.